docs/installation.md

```diff
@@ -133,6 +133,13 @@ sudo PIPX_HOME=/opt/pipx PIPX_BIN_DIR=/usr/local/bin PIPX_MAN_DIR=/usr/local/sha
 # Example: $ sudo PIPX_HOME=/opt/pipx PIPX_BIN_DIR=/usr/local/bin PIPX_MAN_DIR=/usr/local/share/man pipx install cowsay
 ```
 
+### Global installation
+
+Pipx also comes with a `--global` argument which helps to execute actions globally to give app access to any user. By
+default the global binary location is set to `/usr/local/bin`, can be overridden with `PIPX_GLOBAL_BIN_DIR`. Default
+global manual page location is `/usr/local/share/man`. This can be overridden with `PIPX_GLOBAL_MAN_DIR`. Default global
+virtual environment location is `/opt/pipx`, can be overridden with `PIPX_GLOBAL_HOME`.
```

Comment:
The previous paragraph specifies for each of the env variables that they are env variables. While quite verbose, I think it makes it easier to understand for a new user. If you agree, please adjust this new paragraph accordingly.

Reply:
Yes, being new user-friendly is important. I adjusted the paragraph :)